changelog.txt

```diff
@@ -72,6 +72,7 @@ Template for new versions:
 - `exterminate`: show descriptive names for the listed races in addition to their IDs
 - `exterminate`: show actual names for unique creatures such as forgotten beasts and titans
 - `fix/ownership`: now also checks and fixes room ownership links
+- `idle-crafting`: also support making shell crafts for workshops with linked input stockpiles
```

Comment:
should be moved to "Future" section above

Reply:
Oops :)